utils/fo-installdeps

```diff
@@ -157,13 +157,13 @@ if [[ $RUNTIME ]]; then
             dpkg-dev heirloom-mailx
          case "$CODENAME" in
             trusty)
-               apt-get $YesOpt install postgresql-9.3 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
+               apt-get $YesOpt install postgresql-9.3 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl libboost-program-options1.54.0 libboost-regex1.54.0;;
             jessie)
-               apt-get $YesOpt install postgresql-9.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
+               apt-get $YesOpt install postgresql-9.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl libboost-program-options1.55.0 libboost-regex1.55.0;;
             xenial)
-               apt-get $YesOpt install postgresql-9.5 php7.0 libapache2-mod-php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-mbstring php7.0-zip;;
+               apt-get $YesOpt install postgresql-9.5 php7.0 libapache2-mod-php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-zip libboost-program-options1.58.0 libboost-regex1.58.0;;
             stretch|buster|sid)
-               apt-get $YesOpt install postgresql-9.6 php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-xml php7.0-zip;;
+               apt-get "$YesOpt" install postgresql-9.6 php7.0 php7.0-pgsql php7.0-cli php7.0-curl php7.0-xml php7.0-zip libboost-program-options1.62.0 libboost-regex1.62.0;;
```

Comment:
Please remove the quotes. @GMishx

Reply:
Thank you for pointing that out. Please check